core/templates/pages/skill-editor-page/editor-tab/skill-prerequisite-skills-editor/skill-prerequisite-skills-editor.component.spec.ts

```diff
@@ -339,6 +347,9 @@ describe('Skill editor main tab Component', () => {
 
     let desc2 = component.getSkillDescription('BBB6dzfb5pPt');
     expect(desc2).toEqual('Dummy Skill 1');
+
+    let desc3 = component.getSkillDescription(null);
```

Comment:
It looks like this is causing a TS error (`null` can't be assigned to `string` type). But I'm wondering why this test is necessary in the first place? Looking at the template for this component, I can see that this function will only be called if skillId (the argument) is a truthy value, meaning it will never be called with `null`.

I don't think this test is necessary? (@gp201 PTAL)

Reply:
@nithinrdy coverage tests were failing, that's why I added this test. If it is never called with `null`, that line in typescript file can be removed? as well as this test also!
@gp201 please confirm!

Reply:
From the component, we can see that the function does return null if the skill is not found. But I don't think passing a `null` value is the correct approach to test it.
https://github.com/oppia/oppia/blob/84d35d266e0a82fceeb525468bdceff4afcb4226/core/templates/pages/skill-editor-page/editor-tab/skill-prerequisite-skills-editor/skill-prerequisite-skills-editor.component.ts#L128-L135

@Shivkant-Chauhan can you try testing with a real-world value?

Reply:
done this.